README.md

```diff
@@ -78,18 +78,19 @@ mvn clean package -Drevision=$(git describe --tags --always)
+|----------------------------------------|----------|----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `aws.eventbridge.connector.id`         | **Yes**  |                            | The unique ID of this connector (used in the EventBridge event `source` field as a suffix on `kafka-connect.` to uniquely identify a connector).                                                                                                                                                                                                             |
+| `aws.eventbridge.region`               | **Yes**  |                            | The AWS region of the target event bus.                                                                                                                                                                                                                                                                                                                      |
+| `aws.eventbridge.endpoint.uri`         | No       |                            | An optional [service endpoint](https://docs.aws.amazon.com/general/latest/gr/ev.html) URI used to connect to EventBridge.                                                                                                                                                                                                                                    |
+| `aws.eventbridge.eventbus.arn`         | **Yes**  |                            | The ARN of the target event bus.                                                                                                                                                                                                                                                                                                                             |
+| `aws.eventbridge.eventbus.endpoint.id` | No       |                            | An optional global endpoint ID of the target event bus specified using `abcde.xyz` syntax (see API [documentation](https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEvents.html#eventbridge-PutEvents-request-EndpointId)).                                                                                                                                                                                                                                                                                                      |
```

Comment:
Would also link "global endpoint" to https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-global-endpoints.html